src/standalone.ts

```diff
@@ -60,7 +60,14 @@ if (shouldPolyfill()) {
     'src',
     'https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/7.7.0/polyfill.min.js'
   )
-  document.body.appendChild(script)
+
+  if (document.readyState === 'loading') {
+    document.addEventListener('DOMContentLoaded', () =>
+      document.body.appendChild(script)
+    )
+  } else {
+    document.body.appendChild(script)
+  }
```

Comment:
is there a difference if we just do  this?
```js
    document.addEventListener('DOMContentLoaded', () =>
      document.body.appendChild(script)
    )
```

Reply:
Yeah, the event does not trigger retroactively